packages/dashboard/tslint.json

```diff
@@ -0,0 +1,3 @@
+{
+  "extends": ["@uifabric/tslint-rules"]
+}
```

Comment:
The pre-commit hooks were complaining about `tslint` rules not being set up for the `dashboard` packages.